source/JABHandler.py

```diff
@@ -701,37 +701,40 @@ def internal_hasFocus(sourceContext):
 @AccessBridge_PropertyNameChangeFP
 def event_nameChange(vmID,event,source,oldVal,newVal):

```

Comment:
Can modified functions get types added to their header, where the types are known?